guidelines.md

```diff
@@ -34,7 +34,16 @@ The API accepts only `HTTP POST` requests with `Content-Type` set to `applicatio
 | `LanguageCode` | string | optional | Code of the [language](operations/configuration.md#language). |
 | `CultureCode` | string | optional | Code of the culture. |
 
-All operations of the API require `ClientToken` and `AccessToken` to be present in the request. The `ClientToken` serves as an identificator of the client using the API. The `AccessToken` grants the client access to data of an enterprise in the system. For development purposes, use the demo environment. For production usage, the `ClientToken` will be provided to you by our integration team and `AccessToken` by admin of the enterprise. See [Authentication](guidelines.md#authentication) for further details.
+All operations of the API require a `ClientToken` and an `AccessToken` to be present in the request. The `ClientToken` serves as a unique identifier of the integration partner consuming the API. A unique `AccessToken` is generated for each new property using the connection and allows the partner to access the data of that enterprise.
+
+For development and testing of your integration, use the demo environment credentials listed in the [Authentication](guidelines.md#authentication) section below. 
+
+In order to receive credentials for production usage, you will have to successfully complete a certification process. Once certified, our Marketplace team will create your integration profile and you will automatically receive a unique `ClientToken`. This `ClientToken` will stay the same for all of the connections that you configure in the production environment. 
+
+A unique `AccessToken` will automatically be generated for each enterprise requesting to connect their Mews profile to your system. Once the enterprise requests the connection from the Mews Marketplace, the token will automatically be shared with you. Each enterprise will have a different `AccessToken` token and there is no way to simultaneously access multiple properties, even if when in the same Mews chain.
```

Comment:
I would not mention the Marketplace. It can be confusing if they don't know that it is. This is mainly for developers.

Reply:
The original version of the doc states : "Will be provided to you by our integration team upon certification completion. For further information, please contact marketplace@mewssystems.com." -> As their solution is displayed in the Marketplace and we use the marketplace email I thought it would make more sense to call the team the Marketplace team as well for consistency.

Reply:
```
A unique `AccessToken` will automatically be generated for each enterprise requesting to connect their Mews profile to your system. Once the enterprise requests the connection from the Mews Marketplace, the token will automatically be shared with you.
``` 
These two sentences says roughly the same, I'd do something like this instead: 
```
A unique `AccessToken` will automatically be generated and shared with you for each enterprise requesting to connect their Mews profile to your system.
```